Datadog.Trace.Debugger.slnf

```diff
@@ -15,7 +15,8 @@
       "tracer\\test\\Datadog.Trace.ClrProfiler.Managed.Tests\\Datadog.Trace.ClrProfiler.Managed.Tests.csproj",
       "tracer\\test\\Datadog.Trace.TestHelpers\\Datadog.Trace.TestHelpers.csproj",
       "tracer\\test\\Datadog.Trace.Tests\\Datadog.Trace.Tests.csproj",
-      "tracer\\test\\test-applications\\debugger\\Samples.Probes\\Samples.Probes.csproj"
+      "tracer\\test\\test-applications\\debugger\\Samples.Probes\\Samples.Probes.csproj",
+	  "tracer\\test\\test-applications\\debugger\\dependency-libs\\Samples.Probes.External\\Samples.Probes.External.csproj"
```

Comment:
Nitpick: fix indentation

Reply:
That's Github's illusion - there is no indentation mismatch.

Reply:
Are you sure? Tabs vs. spaces?

Reply:
No biggie either way, feel free to resolve.

Reply:
You're right, it was tabs/spaces difference. Fixed in 4a24197b933cf14420bcf94fa948fd875cc8651a.